ENIGMAsystem/SHELL/Widget_Systems/xlib/dialogs.cpp

```diff
@@ -87,6 +90,19 @@ static bool kwin_running() {
   return bKWinRunning;
 }
 
+static unsigned long *widget_icon = nullptr;
```

Comment:
Safer to make this a unique_ptr and use `reset(bgra_to_argb(...))` and `widget_icon.get()`, below